week06_style_transfer/style_transfer_pytorch.ipynb

```diff
@@ -26,6 +26,18 @@
     "# !pip3 install --user torchvision"
```

Comment:
We are making some random changes
<br/>

 _Reply via <a href='https://app.reviewnb.com/test-v1/Practical_DL/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>